script.js

```diff
@@ -0,0 +1,97 @@
+class Books {
+  constructor(title = 'Lorem ipsum', author = 'Testero Testy') {
+    this.booktitle = title;
+    this.bookauthor = author;
+  }
+
+  set bookTitle(title) {
+    this.booktitle = title;
+  }
+
+  set bookAuthor(author) {
+    this.bookauthor = author;
+  }
+
+  get bookTitle() {
+    return this.booktitle;
+  }
+
+  get bookAuthor() {
+    return this.bookauthor;
+  }
+
+  toPlainObject() {
+    return { title: this.booktitle, author: this.bookauthor };
+  }
+
+  static toClassObject(obj) {
+    return new Books(obj.title, obj.author);
+  }
+}
+
+function parseString() {
+  const localstorageData = JSON.parse(localStorage.getItem('data'));
+  const localstorageDataObj = localstorageData.map((booksData) => Books.toClassObject(booksData));
+  return localstorageDataObj;
+}
+
+// let booksobjects = [];
+const booksobjects = parseString();
+
+const displayBooks = () => {
+  let bookData = '';
+  for (let i = 0; i < booksobjects.length; i += 1) {
+    bookData += `<p>${booksobjects[i].bookTitle}</p>
+        <p>${booksobjects[i].bookAuthor}</p>
+        <button class="remove-book" onclick="removeBook(${i})">Remove</button>
+        <hr>`;
+  }
```

Comment:
Hi @muhammaduzairrazaq 🔢 
Good job so far but I am afraid your onClick event is not working properly. When I tried to add a book in you app the Add button is not working at all. Please fix the potential bug:
![bug](https://github.com/muhammaduzairrazaq/awesomeBooks/assets/122353030/c26d31ab-8166-4935-877b-aad6dc5cde24)